### ISSUE TITLE
VVSG2 Limited Dexterity Accessibility

### ISSUE THREAD

**Issue by mattroe** [votingworks/vxsuite-roadmap#22]
**What is the feature or problem to solve?**
A voter with limited dexterity needs to able to mark and cast their ballot privately and independently. 

**Why is it important?**
We cannot meet VVSG2 accessibility requirements without solving this problem.

**Comment by mattroe** [comment]
Sip & puff experience examples:
* ES&S: https://www.youtube.com/watch?v=nzij-DMVkfA
* ClearBallot: https://www.youtube.com/watch?v=8C4ftz50cpE
* Dominion: https://www.youtube.com/watch?v=ROFGnQqJpeQ
